.github/workflows/lint.yml

```diff
@@ -19,6 +19,8 @@ jobs:
   lint:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
+    # Only run on origin repository not forked repository
+    if: github.repository == 'web-infra-dev/rspress'
```

Comment:
We can still run the lint CI in forked repo, it helps developer to test their changes.

Reply:
ok